test/array.test.js

```diff
@@ -1,37 +1,37 @@
 import { ArrayColl } from '..';
 
 function newArray() {
-  let a = new ArrayColl();
-  a.add("a");
-  a.add("b");
-  a.add("c");
-  a.add("c");
-  a.add("d");
-  return a;
+  return ArrayColl.from(["a", "b", "c", "c", "d"]);
 }
 
 test('Array add, remove', () => {
   let a = newArray();
   expect(a.length).toBe(5);
 
-  a.push("d"); // alias for add()
+  a.push("d", "d"); // alias for addAll([...])
```

Comment:
Please keep a single-param version of the function call in the test. You can add a multi-param version, but the most simple version of the usage should also be covered in the test.